### ISSUE TITLE
Please tag releases

### ISSUE THREAD

**Issue by onlyjob** [yacy/yacy_search_server#348]
It would be useful to tag source releases here to sync with https://yacy.net/download_installation/

Thanks.

**Comment by sumdog** [comment]
I agree. I'm working on trying to package Yacy for Nix and there's no tag for `yacy_v1.924_20210209_10069`. Which commit is that?  Is it b46513f4a16f36d34f23c4b7dd07e38919d3fafe? That's the last one made on February 9th, 2021, but there's also a CVE commit right after that on 22 Feb! 

Does the current version for download have an open security vulnerability?

**Comment by okybaca** [comment]
hope it's fixed, as well as #665
